src/main/java/de/teamlapen/vampirism/entity/EntityVampirism.java

```diff
@@ -33,6 +37,9 @@
     private final EntityAIBase moveTowardsRestriction;
     protected boolean hasArms = true;
     protected boolean peaceful = false;
+    /** available actions for AI task & task */
+    protected EntityActionHandler<?> entityActionHandler;
+    protected List<IEntityAction> entityActions;
```

Comment:
Maybe we can get rid of this variable now and pass the values directly to the action handler